railties/lib/rails/commands/server/server_command.rb

```diff
@@ -161,7 +162,8 @@ def server_options
             daemonize:             options[:daemon],
             pid:                   pid,
             caching:               options["dev-caching"],
-            restart_cmd:           restart_command
+            restart_cmd:           restart_command,
+            early_hints:           early_hints
```

Comment:
use it directly like dev-caching / daemon ?

Reply:
I put the documentation in the early hints option. It's passed in like `--early-hints` or can be set in the puma.config with `early_hints(true)`.